### PR TITLE
Tried a few things but settled on this, simplified welcome + button, …

### PR DESCRIPTION
**Pull request by donalod**
Tried a few things but settled on this, simplified welcome + button, does the job, front and centre

### CODE REVIEW

File: pub/index.html [hunk 1]
```diff
@@ -60,19 +60,19 @@
...
-         <div class="MeetupBox">
-          <p><strong>iNOG::D</strong></p>
-          <p>Wed 20 Sep 2017
-          <br>6:00 PM to 9:30 PM
-          <br>Workday, Dublin</p>
-          <p><a href="https://www.meetup.com/Irish-Network-Operators-Group/events/228394358/" target="_blank" class="btn btn-warning"><strong>RSVP on Meetup!</strong></a></p>
-          <p><small class="futuremeets"></small></p>
-         </div>
+        <div class="col-lg-9 col-md-12 col-sm-12 col-xs-12">
+          <h1 style="color: white;" >Hello, Ireland!</h1>
```
Comment (cmsirbu) [inline pub/index.html]: It's white by default, why is this needed?

File: pub/index.html [hunk 2]
```diff
@@ -60,19 +60,19 @@
...
-        <div class="col-lg-3">
-         <div class="MeetupBox">
-          <p><strong>iNOG::D</strong></p>
-          <p>Wed 20 Sep 2017
-          <br>6:00 PM to 9:30 PM
-          <br>Workday, Dublin</p>
-          <p><a href="https://www.meetup.com/Irish-Network-Operators-Group/events/228394358/" target="_blank" class="btn btn-warning"><strong>RSVP on Meetup!</strong></a></p>
-          <p><small class="futuremeets"></small></p>
-         </div>
+        <div class="col-lg-9 col-md-12 col-sm-12 col-xs-12">
```
Comment (cmsirbu) [inline pub/index.html]: Let's keep col-md-9 with 3 so that in medium resolutions it still looks as designed. I'd like to avoid the collapsed view on PC which will happen on smaller screens/older laptops etc.

File: pub/index.html [hunk 3]
```diff
@@ -60,19 +60,19 @@
...
+          <h1 style="color: white;" >Hello, Ireland!</h1>
+          <p>Help us build an inclusive and open <strong>Community of Practice</strong> around network engineering. The Irish Network Operators Group (iNOG) is re-birthing community focused on diversity, learning, sharing, connecting, and having fun. &nbsp;&nbsp;&nbsp;<a class="btn btn-sm btn-success" data-toggle="modal" data-target="#CofCmodal">Code of Conduct</a></p>
```
Comment (cmsirbu) [inline pub/index.html]: If you replace the " &nbsp;&nbsp;&nbsp;" with a <br> then it looks/flows a lot better. That button is so unaligned otherwise it hurts :)

File: pub/index.html (hunk 1 above)
Comment (donalod) [reply pub/index.html]: Left over from trying other heading sizes, removed.

File: pub/index.html (hunk 2 above)
Comment (donalod) [reply pub/index.html]: Done.

File: pub/index.html (hunk 3 above)
Comment (donalod) [reply pub/index.html]: Sure.
